tt_eager/tt_dnn/op_library/bmm/kernels/dataflow/reader_bmm_tile_layout_in0_sender_receiver_padding_block_sharded.cpp

```diff
@@ -105,7 +140,6 @@ void kernel_main() {
                 // num_dests must not include source, since we are NOT really doing a local copy!
                 // noc_semaphore_set_multicast(in0_mcast_receiver_semaphore_addr, in0_mcast_receiver_semaphore_noc_addr, in0_mcast_num_cores);
                 noc_semaphore_set_multicast_loopback_src(in0_mcast_receiver_semaphore_addr, in0_mcast_receiver_semaphore_noc_addr, in0_mcast_num_cores+1, false);
-                local_read_addr += in0_block_size_bytes;
```

Comment:
Is this safe to remove for the old cases? I think probably, but it's there to generalize to batches, which we don't have yet. I would leave this in.